README.md

```diff
@@ -16,10 +16,33 @@ staramr search -o out --pointfinder-organism salmonella *.fasta
 
 **out/summary.tsv**:
 
-| Isolate ID | Genotype                                                  | Predicted Phenotype                                                                                       |
-|------------|-----------------------------------------------------------|-----------------------------------------------------------------------------------------------------------|
-| SRR1952908 | aadA1, aadA2, blaTEM-57, cmlA1, gyrA (S83Y), sul3, tet(A) | streptomycin, ampicillin, chloramphenicol, ciprofloxacin I/R, nalidixic acid, sulfisoxazole, tetracycline |
-| SRR1952926 | blaTEM-57, gyrA (S83Y), tet(A)                            | ampicillin, ciprofloxacin I/R, nalidixic acid, tetracycline                                               |
+| Isolate ID | Genotype                                                  | Predicted Phenotype                                                                                       | Plasmid Genes                       |
```

Comment:
Thanks for spending the time to update all of these :smile: 

Could you maybe cut each table in these examples to 2 or 3 rows? These are only meant to be example outputs and this way people won't have to scroll so much to get to the table of contents

Reply:
Will do, changed it to 2 rows now